frontend/src/scenes/constructor/sagas/index.ts

```diff
@@ -0,0 +1,75 @@
+import { all, call, put, takeEvery } from 'redux-saga/effects';
+import { Routine } from 'redux-saga-routines';
+import {
+  deleteSelectQueryRoutine,
+  duplicateSelectQueryRoutine,
+  fetchQueryRoutine, openQueryRoutine,
+  saveSelectQueryRoutine
+} from '../routines';
+import { IQuery } from '../../../common/models/apps/querys';
+import { addQuery, deleteQuery, fetchQueries, updateQuery } from '../../../services/queryService';
+
+function* fetchQuery({ payload }: Routine<any>) {
+  try {
+    const queries:Array<IQuery> = yield call(fetchQueries, payload.id);
+    console.log(queries);
```

Comment:
Don't use console.log, remove it everywhere. There should be no eslint warnings or your build will fail